src/test/java/com/solvd/carina_testing/MobileTesting.java

```diff
@@ -0,0 +1,38 @@
+package com.solvd.carina_testing;
+
+import com.zebrunner.carina.core.AbstractTest;
+import com.zebrunner.carina.utils.mobile.IMobileUtils;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class MobileTesting extends AbstractTest implements IMobileUtils {
+
+    private static final String TITLE_TASK = "Go to the shop";
+    private static final  int INDEX_TASK = 2;
+
+    @Test
+    public void verifyCreatingTask(){
+        TodayPage todayPage =  new TodayPage(getDriver());
+        startApp("com.todoist");
+        int numOfTasksBeforeAddNew = todayPage.getTaskCards().size();
+        todayPage.addNewTask();
+        Assert.assertTrue(todayPage.isAirplaneButtonPresent(), "Airplane button isn`t present!");
+        todayPage.createNewTask(TITLE_TASK);
+        int numOfTasksAfterAddNew = todayPage.getTaskCards().size();
+        System.out.println(numOfTasksBeforeAddNew + " " + numOfTasksAfterAddNew);
+        Assert.assertTrue(numOfTasksAfterAddNew > numOfTasksBeforeAddNew, "New task was not added");
```

Comment:
actually that is ordinary verification, we use it in real work. In your case check that it's enough time for loading page after adding new task for collecting tasks to the list, check if keyboard hides. If there is a way, you can check if task by name present (create locator for tasks' name and set expected name to it)